state-chain/pallets/cf-ingress-egress/src/mocks.rs

```diff
@@ -148,6 +149,7 @@ impl Config<Instance2> for Test {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
 	const MANAGE_CHANNEL_LIFETIME: bool = true;
+	const CAN_PREALLOCATE_ADDRESSES: bool = true;
```

Comment:
This name doesn't really reflect what's happening, since we end up preallocating for all chains. 
Something like `OnlyPreallocateFromPool` perhaps? Where Ethereum and Arbitrum would then be true (and it clearly means we don't preallocate from elsewhere)
- open to other names, but the current one is inaccurate